### PR TITLE
[v7r2] fixes for ES docs, SummarizeLogsAgent and StatesAccountingAgent can also commit to Monitoring

### PR DESCRIPTION
**Pull request by fstagni**
The change to StatesAccountingAgent can/should be advertised in th wiki, if agreed.

BEGINRELEASENOTES

*WMS
NEW: moving towards a single agent for WMS history

*docs
FIX: added config and instructions for Monitoring DBs (ES) configuration

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/WorkloadManagementSystem/Agent/StatesAccountingAgent.py [hunk 1]
```diff
@@ -37,23 +35,32 @@ class StatesAccountingAgent(AgentModule):
...
 
   def initialize(self):
-    """ Standard constructor
+    """ Standard initialization
     """
-    self.dsClients = {}
-    self.jobDB = JobDB()
-    self.retryOnce = False
-    self.retryValues = []
+    # This agent will always loop every 15 minutes
```
Comment (atsareg) [inline src/DIRAC/WorkloadManagementSystem/Agent/StatesAccountingAgent.py:49]: Why is this hard coded ?

Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/Agent/StatesAccountingAgent.py:49]: It was hard-coded also before...

**Review by chrisburr — DISMISSED** [review]

**Comment by fstagni** [comment]
Review, please.

File: docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst [hunk 2]
```diff
@@ -47,18 +48,48 @@ For example::
...
 
+The given periods above are also the default periods in the code.
+
 
 Enable WMSHistory monitoring
 ============================
 
-You have to install the WorkloadManagemet/StatesMonitoringAgent. This agent is used to collect information using the JobDB and send it to the Elasticsearch database.
-If you install this agent, you can stop the StatesAccounting agent.
+You have to install the :mod:`~DIRAC.WorkloadManagementSystem.Agent.StatesMonitoringAgent`.
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst:87]: So this isn't true anymore, isn't it? One doesn't have to install this agent, instead one can install the StatesAccountingAgent and set Backends=Monitoring

**Comment by atsareg** [comment]
You can also add to the doc the case of ES using with certificate based login as described in #4924

**Comment by fstagni** [comment]
Well that should be added to v7r1. I created https://github.com/DIRACGrid/DIRAC/issues/4940 from your comment above.

**Review by atsareg — APPROVED** [review]

**Pull request merged**